cluster/addons/cluster-loadbalancing/glbc/default-controller.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: glbc-default-backend
```

Comment:
yeah, did that only downside is i now need to modify docs to contain a -c but meh